src/lib/structures/JoshProvider.ts

```diff
@@ -6,12 +6,9 @@ import type { Josh } from './Josh';
 
 /**
  * The base provider class. Extend this class to create your own provider.
- *
- * NOTE: If you want an example of how to use this class please see `src/lib/structures/default-provider/MapProvider.ts`
- *
- * @see {@link JoshProvider.Options} for all options available to the JoshProvider class.
- *
  * @since 2.0.0
+ * @see [MapProvider](default-provider/MapProvider.ts) for an example of how to use this class.
+ * @see {@link JoshProvider.Options} for all options available to the JoshProvider class.
```

Comment:
This will be changed later on when `MapProvider` is moved to `/providers`, for now we can just undo this change.

Reply:
Ah okie dokie!

Reply:
This change still needs to be undone.

Reply:
Hopefully https://github.com/josh-development/core/pull/70/commits/da887158491aee775b77072e04b3c3c81e2a4ec0 is right? I still kept the `@since 2.0.0` and `@see {@link JoshProvider.Options}...` below the description for the standard.